### DISCUSSION TITLE
Sankey website down?

### DISCUSSION THREAD

**Discussion by PNRYates** [firefly-iii/firefly-iii#8875] [Firefly III]
the sankey api project's website [[(link)](https://github.com/firefly-iii/sankey)](https://github.com/firefly-iii/sankey) doesn't seem to work anymore.

**Comment by JC5** [comment]
That's correct. It's been down for about a year now. I'll update the repository. The code, meanwhile, has been made part of the v2 layout.

**Reply by tmeekes** [reply]
~~Just to get clarity on this... does this mean you can create sankey diagrams in firefly with v2?~~

Nevermind, loaded up v2 again and I see it now, on the bottom of the home page.
